command/flags.go

```diff
@@ -113,6 +111,26 @@ var InitFlags = []cli.Flag{
 	},
 }
 
+var RegisterFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:     "config",
+		Usage:    "Config file containing provider's peer ID and private key",
+		Required: true,
+	},
+	&cli.StringFlag{
+		Name:     "indexer-addr",
+		Usage:    "Indexer address:port to register at",
```

Comment:
maybe call this the 'control port' of the indexer?

Reply:
Well... Need the address too because this could be talking to an indexer running somewhere else.

This is more a simple client than a command.